app/controllers/concerns/hyrax/contact_form_recaptcha_behavior.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Hyrax
+  # Behavior for recaptcha
+  module ContactFormRecaptchaBehavior
+    def check_recaptcha
+      if recaptcha?
```

Comment:
You shouldn't need to override the method length rubocop, refactor the method along the lines of

```
return true unless recaptcha? && verify_recaptcha(model: @contact_form)
flash.now[:error] = 'Captcha did not verify properly.'
false
```